RockLib.Compression.Tests/RockLib.Compression.Tests.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netcoreapp3.1;net462;</TargetFrameworks>
+    <TargetFrameworks>net6.0;netcoreapp3.1;net48;</TargetFrameworks>
```

Comment:
does this have any impact on what version of dotnet the consumer needs to be using? if so, that should probably be documented in the README

Reply:
(well this is the test file, but i think the main proj is updated too :) )

Reply:
updated.